robot/server/server.go

```diff
@@ -551,6 +551,21 @@ func (s *Server) GetMachineStatus(ctx context.Context, _ *pb.GetMachineStatusReq
 		result.State = pb.GetMachineStatusResponse_STATE_RUNNING
 	}
 
+	if s.robot.JobManager() != nil {
+		if n := s.robot.JobManager().NumJobHistories.Load(); n > 0 {
```

Comment:
if we have MachineStatus compute this, we wouldn't need to add a new method to the robot interface, right?